jenkins/check-for-build.jenkinsfile

```diff
@@ -14,6 +14,9 @@ pipeline {
     }
     triggers {
         parameterizedCron '''
+            H 1 * * * %INPUT_MANIFEST=2.5.0/opensearch-dashboards-2.5.0.yml;TARGET_JOB_NAME=distribution-build-opensearch-dashboards;BUILD_PLATFORM=linux windows
+            H 1 * * * %INPUT_MANIFEST=2.4.1/opensearch-dashboards-2.4.1.yml;TARGET_JOB_NAME=distribution-build-opensearch-dashboards;BUILD_PLATFORM=linux windows
+            H 1 * * * %INPUT_MANIFEST=2.4.1/opensearch-2.4.1.yml;TARGET_JOB_NAME=distribution-build-opensearch;BUILD_PLATFORM=linux macos windows
```

Comment:
Nice to have these platform params added 😄

Reply:
Didn't we default only to linux in your PR?

Reply:
@gaiksaya No. It adds `linux macos windows` for `opensearch` and `inux windows` for OSD at this point of time as we plan to support windows for 1.x in future. It would be default to just `linux` if it's not `opensearch` nor `opensearch-dashboards` which shouldn't happen right now.
We already have cron for `1.3.7` below so for the workflow will add with this pattern for `1.3.8` once it's incremented.

Reply:
Until it is finalized I believe windows should be removed. But should be fine for now.